src/components/question/index.cjsx

```diff
@@ -11,7 +11,6 @@ QuestionHtml = React.createClass
   propTypes:
     html: React.PropTypes.string
     type: React.PropTypes.string
-    questionNumber: React.PropTypes.number
```

Comment:
I get the sense that these `questionNumber` related thing in this file are needed?  They were added in this recent PR #95